.github/workflows/raspberry_pi.yml

```diff
@@ -0,0 +1,22 @@
+name: Raspberry Pi
+on:
+  workflow_dispatch:
+    inputs:
+      extra_resolve_options:
+        description: "Extra Resolve Options"
+        required: false
+  schedule:
+    - cron: "0 1 * * *" # 3 AM CET
+  push:
+  pull_request:
+jobs:
+  RaspberryPi:
+    uses: steinwurf/cross-compile-action/.github/workflows/action.yml@3.0.0
+    with:
+      extra_resolve_options: ${{ github.event.inputs.extra_resolve_options }}
+      name: Raspberry Pi 3
+      cxx_mkspec: cxx_raspberry_gxx83_armv7
+      image: ghcr.io/steinwurf/gcc-8.3.0-raspberrypi-armv7:1.0.0
+      test_runner: raspberry_pi_3
```

Comment:
I think it would be best to just use raspberry_pi here since I don't think we really care about the version here? What do you think?

Reply:
Yes, agreed.